typescript/cli/src/context/context.ts

```diff
@@ -166,7 +185,7 @@ function getRegistry(
  */
 async function getMultiProvider(registry: IRegistry, signer?: ethers.Signer) {
   const chainMetadata = await registry.getMetadata();
-  const multiProvider = new MultiProvider(chainMetadata);
+  const multiProvider = new MultiProvider(chainMetadata as any);
```

Comment:
Why is this cast necessary now? If it's because the registry repo's SDK and the CLI's SDK versions don't match up, let's update the registry

Reply:
this should not have been committed

Reply:
Oh wow okay this is using the old version of the Registry. Will need to up this when we merge the new registry version

Reply:
Upped to latest registry